index.html

```diff
@@ -253,6 +254,39 @@ <h3>A Special Time</h3>
             </div>
           </div>
       </article>
+      <h3 style="max-width: 600px; margin: 0 auto;" class="formHeader">Send us a messsage</h3>
+      <form>
+        <div class="formContainer">
+          <div class="formQuestion">
+            <label for="name">Name </label>
+            <input type="text" name="user_name" id="name" required>
+          </div>
+          <div class="formQuestion">
+            <label for="email">Email </label>
+            <input type="email" name="user_email" id="email" required>
+          </div>
+          <div class="formQuestion">
```

Comment:
clearly identified a class is a powerful tool. Instead of naming that class formQuestion I suggested to name formInput

Reply:
I think "formInput" should be used to target only the input. I used "formQuestion'" because there was a label together.